testData/solutions/win32.json5

```diff
@@ -1691,396 +1773,434 @@
                                 2
                             ],
                             "enumLabels": [
-                                "line",
-                                "sentence",
-                                "paragraph"
+                                "Beginner, Highest",
+                                "Intermediate",
+                                "Advanced, Lowest"
                             ]
                         }
                     },
-                    // Requires creating a schema or knowing the name of the default.  Leave out for now.
-                    // "Options.SayAllScheme": {
-                    //     "schema": {
-                    //         "title": "\"Say All\" Scheme",
-                    //         "description": "Which user-defined \"Say All\" scheme to use.",
-                    //         "type": "string"
-                    //     }
-                    // },
-                    "Options.SayBlankLineCount": {
+
+                    // ---- Speech Verbosity: Configure Verbosity Levels -------
+
+                    // NOTE: This settings have the following format:
+                    // =========================================================
+
+                    //  - "1|2|2|Help Balloon"
```

Comment:
This is bordering on the dangerous complexity of their bitwise fields.  IMO, it should either be broken apart into a few "aliases" and transformed into the combined value, or just commented out.

Reply:
These have all been commented out for the meantime.